packages/xo-web/src/common/xoa-updater.js

```diff
@@ -35,6 +35,73 @@ export function blockXoaAccess(xoaState) {
   return block
 }
 
+export function getLicenseNearExpiration(licenses) {
```

Comment:
```suggestion
export function getLicensesNearExpiration(licenses) {
```

are you searching for several or just one?

Reply:
this return only one